modules/govuk_ci/manifests/agent/docker.pp

```diff
@@ -2,6 +2,12 @@
 #
 # Installs and configures Docker and Docker Compose
 class govuk_ci::agent::docker {
-  include ::docker
```

Comment:
It has the same meaning as the `class { '::docker': ... }` definition below but that also provides options (ie `docker_users`)